tests/testthat/test-kwic.R

```diff
@@ -396,3 +396,32 @@ test_that("deprecated keywords argument still works", {
         "fox"
     )
 })
+
+test_that("test docvar is passed through kwic()", {
+    
+    # kwic only copy original docvars
+    testkwic <- kwic(data_corpus_inaugural, 'american')
+    expect_equal(attr(testkwic, 'docvars'),
```

Comment:
If we do attach docvars in this way to kwic objects, should there be a `docvars.kwic()`, or are you just storing the docvars in order to get them when you create `as.tokens()`?